src/hooks/useFetchChannel.jsx

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const useFetchChannel = (channelId) => {
+  const [channelInfo, setChannelInfo] = useState({});
```

Comment:
```suggestion
  const [channelInfo, setChannelInfo] = useState(null);
```
`channelInfo`의 초기값을 `{ }` 대신 `null`로 설정하는 건 어떨까요??

지금은 빈 객체로 초기화하고 있는데, 초기값을 `null`로 두면 `channelInfo`가 있는지 여부를 더 명확히 판단할 수 있어서 가독성이 좋아질 것 같아요! 
@faber-h 희주님 생각은 어떠신지 궁금합니다😊

Reply:
혹시라도 잘못된 반환 값으로 빈 객체가 들어올 경우를 방지할 수 있을 것 같아
이번 `channelInfo`의 경우에는 초기 값을 null로 설정하는 것이 좋을 것 같아요 😀

Reply:
서버로부터 받아오는 데이터가 객체라 해당 상태가 객체가 될 것임을 알려주고 싶어 초기값을 `{}`로 설정했는데 빈 객체는 데이터를 받아왔는데 해당 데이터가 없어서 빈 객체인건지 혼동을 줄 수도 있겠네요. `null`이 더 좋은 것 같습니다. 두 분 좋은 의견 감사드립니다!